### PR TITLE
Improvements for the migration guide

### PR DESCRIPTION
**Pull request by tsg**
Added a section about the changes to the output fields and a few other
clarifications/fixes.

### CODE REVIEW

**Comment by andrewkroh** [comment]
LGTM.

File: docs/migration.asciidoc [hunk 1]
```diff
@@ -90,7 +90,10 @@ with this configuration in Logstash Forwarder:
       ],
 
       # A dictionary of fields to annotate on each event.
-      "fields": { "type": "syslog" }
+      "fields": {
+        "type": "syslog",
+        "service": "example-service"
```
Comment (ruflin) [inline docs/migration.asciidoc]: I like that you extended this. I was thinking about adding even more fields here to make it clearly differentiate from document_type and input_type. Additional option could be: "zone": "us-east-1". I would also replace "example-service" through apache to make it more specific.

**Comment by ruflin** [comment]
LGTM. I don't like that Appveyor is flaky again. Will look into this.

**Comment by monicasarbu** [comment]
+1

File: docs/migration.asciidoc (hunk 1 above)
Comment (monicasarbu) [inline docs/migration.asciidoc]: I agree "apache" is better than "example-service". It's just easier to understand.

**Pull request merged**